browser/ui/android/strings/android_brave_strings.grd

```diff
@@ -3983,6 +3983,12 @@ If you don't accept this request, VPN will not reconnect and your internet conne
     <message name="IDS_IN_APP_UPDATE_TEXT" desc="Text for in app update snackbar">
       Update is downloaded.
     </message>
+    <message name="IDS_UPDATE_FILTER_LIST_SUCCESS_TEXT" desc="Text to notify users for updating filterlist successfully">
+      Filterlist is updated successfully
+    </message>
+    <message name="IDS_UPDATE_FILTER_LIST_FAIL_TEXT" desc="Text to notify users for the failure of updating filterlist">
+      Filterlist could not be updated. Please try again
+    </message>
```

Comment:
I think on desktop we use `Filter list` as 2 words.

Reply:
Updating now. Thank you @mkarolin